dataland-frontend/src/utils/DataPoint.ts

```diff
@@ -1,5 +1,11 @@
 import { type BaseDocumentReference, type ExtendedDocumentReference, type QualityOptions } from "@clients/backend";
 
+type BaseDocumentReferenceWithComment = {
```

Comment:
Is this one new in the GDV framework?

Reply:
Do we have something corresponding in the backend? And do we really need this comment?

Reply:
There is no longer this type. Was removed after refactoring

Reply:
ok